src/ui/concrete-playback/concretePlayback.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+import process = require('process');
+
+import * as vscode from 'vscode';
+
+import { KaniArguments, KaniConstants } from '../../constants';
+import { checkCargoExist, getRootDir } from '../../utils';
+
+/**
+ * Call the visualize flag on the harness and render the html page
+ *
+ * @param commandURI - vscode command that is being executed
+ * @param harnessObj - metadata about the harness
+ */
+export async function callConcretePlayback(
+	commandURI: string,
+	harnessObj: { harnessName: string; harnessFile: string; harnessType: boolean },
+): Promise<void> {
+	let finalCommand: string = '';
+
+	const platform: NodeJS.Platform = process.platform;
+	const harnessName: string = harnessObj.harnessName;
+	const harnessFile: string = harnessObj.harnessFile;
+	const harnessType: boolean = harnessObj.harnessType;
+
+	// Detect source file
+	const terminal = vscode.window.activeTerminal ?? vscode.window.createTerminal();
+
+	// Generate the final visualize command for the supported platforms
+	if (platform === 'darwin' || platform == 'linux') {
+		const responseObject: string = createCommand(commandURI, harnessFile, harnessName, harnessType);
+		const crateURI: string = getRootDir();
+		finalCommand = `cd ${crateURI} && ${responseObject}`;
+	}
+
+	// Wait for the the visualize command to finish generating the report
+	terminal.sendText(finalCommand);
```

Comment:
What will happen on Windows? Does this code silently ignore the user if they push the button on Windows?

Reply:
This does not refer to the linux terminal directly, it refers to the default terminal that is used by the current system, and wrapped by VSCode. If the user is using the extension on VSCode Windows, then it will take care of the disambiguation and create a command to run on the bash underneath. So it should not be a problem.

Reply:
But if the platform is Windows, then the if statement on line 31 will not be entered, and `finalCommand` will therefore be `''` as it was defined on line 20?

Reply:
Resolved offline, the extension is not supported on Windows